docs/merkletree/treetex/main.go

```diff
@@ -185,16 +192,20 @@ func drawLeaf(prefix string, index uint64) {
 // Returns a func to be called to close the node.
 func openInnerNode(prefix string, id compact.NodeID) func() {
 	attr := nInfo[id].String()
-	fmt.Printf("%s [%d.%d, %s, tier=%d\n", prefix, id.Level, id.Index, attr, id.Index-1)
+	fmt.Printf("%s [%d.%d, %s, tier=%d\n", prefix, id.Level, id.Index, attr, id.Level)
```

Comment:
I saw this change in a different PR. Should it be deleted from this one?

Reply:
yeah, it was just stacked on top - the other one is merged and this one rebased now.